test/integration/scheduler_perf/config/performance-config.yaml

```diff
@@ -1,13 +1,14 @@
-# The following labels are used in this file:
-# - fast: short execution time, ideally less than 30 seconds
-# - integration-test: used to select workloads that
-#   run in ci-kubernetes-integration-master. Choosing those tests
-#   is a tradeoff between code coverage and overall runtime.
+# The following labels are used in this file. (listed in ascending order of the number of covered test cases)
 # - short: used to select workloads that
 #   run in pull-kubernetes-integration.
-#   We should make each test case with short label very small,
+#   We should make each test case with short label very small and also skip some less important test cases,
 #   so that all tests with the label should take less than 5 min to complete.
 #   They can be run using --short=true flag.
+# - integration-test: used to select workloads that
+#   run in ci-kubernetes-integration-master. Choosing those tests
+#   is a tradeoff between code coverage and overall runtime.
+# - fast: used to select workloads that require short execution time, ideally less than 30 seconds.
```

Comment:
TBH, I'm not very sure if we need "fast" in the first place, even after reading [the discussion](https://github.com/kubernetes/kubernetes/pull/126352/files#r1719964207).
Isn't "integration-test" enough to test the change of scheduler-perf? (Have they been too small to find out some bugs in scheduler-perf?)

Reply:
No suite uses the `fast` tag. It's just for developers' convenience, so they don't have to wait forever to run the suite when doing local changes.

I agree that `fast` is confusing. I think it was added before adding `short`. I think it might make more sense to do something like this:

- pull-kubernetes-integration: short + integration
- ci-kubernetes-integration: integration
- developer mode: short + performance
- ci-benchmark-scheduler-perf: performance

Reply:
> Have they been too small to find out some bugs in scheduler-perf?

Yes, `integration-test` labeled workloads can be too small to spot the issues, especially for the cases where you need to run for some time to select a correct code path. 

> * pull-kubernetes-integration: short + integration
> * ci-kubernetes-integration: integration
> * developer mode: short + performance
> * ci-benchmark-scheduler-perf: performance

That looks good. I think we could implement it.

Reply:
Looks pretty. Where do we control which label is used for which CI? 
I could find in https://github.com/kubernetes/test-infra..

Reply:
- `integration-test`: as a default in scheduler_perf when running it as test ([ref](https://github.com/kubernetes/kubernetes/blob/88b93957c79f52e62e8e85f674ef6d7f10a2309b/test/integration/scheduler_perf/scheduler_perf.go#L977))
- `performance`: as a default in scheduler_perf when running it as benchmark ([ref](https://github.com/kubernetes/kubernetes/blob/88b93957c79f52e62e8e85f674ef6d7f10a2309b/test/integration/scheduler_perf/scheduler_perf.go#L815))
- `short`: taken from `go test --short=true/false` flag. Running `make integration-test` set this flag as true by default ([ref](https://github.com/kubernetes/kubernetes/blob/88b93957c79f52e62e8e85f674ef6d7f10a2309b/hack/make-rules/test-integration.sh#L83)), but for periodic CI job it is overwritten to false in test-infra ([ref](https://github.com/kubernetes/test-infra/blob/c01c0fc901ba2821c354e4d0b7e978e29d80696d/config/jobs/kubernetes/sig-testing/integration.yaml#L136)).

Reply:
Thanks! Done.

Reply:
Can you check if short flag enabled by default ([ref](https://github.com/kubernetes/kubernetes/blob/88b93957c79f52e62e8e85f674ef6d7f10a2309b/hack/make-rules/test-integration.sh#L83)) won't break the performance tests in `ci-benchmark-scheduler-perf` (and presubmit invoked by a command)? Perhaps we should explicitly set it to false in test-infra similarly to integration tests.

Reply:
Good point, made a PR:
https://github.com/kubernetes/test-infra/pull/33498

Reply:
/hold

until https://github.com/kubernetes/test-infra/pull/33498